spec.bs

```diff
@@ -406,6 +463,30 @@ A [=component=] has an associated <dfn for=component>group name list</dfn>, an a
   1. Return |result|.
 </div>
 
+The <dfn>default options</dfn> is an [=options=] [=struct=] with [=options/delimiter code point=] set to the empty string and [=options/prefix code point=] set to the empty string.
+
+The <dfn>hostname options</dfn> is an [=options=] [=struct=] with [=options/delimiter code point=] set "`.`" and [=options/prefix code point=] set to the empty string.
+
+The <dfn>standard pathname options</dfn> is an [=options=] [=struct=] with [=options/delimiter code point=] set "`/`" and [=options/prefix code point=] set to "`/`".
+
+<div algorithm>
+  To determine if a <dfn>protocol component matches a special scheme</dfn> given a [=component=] |protocol component|:
+
+  1. Let |special scheme list| be a [=list=] populated with all of the [=special scheme=] values.
+  1. [=list/For each=] |scheme| of |special scheme list|:
+    1. Let |test result| be [$RegExpBuiltinTest$](|protocol component|'s [=component/regular expression=], |scheme|).
```

Comment:
RegExpBuiltinTest doesn't seem to exist. I think you want to use BuiltinExec and check for null.